pyuvdata/utils.pyx

```diff
@@ -162,18 +199,29 @@ cpdef numpy.ndarray[dtype=numpy.int64_t] antnums_to_baseline(
     if less256:
       _antnum_to_bl_256(ant1, ant2, _bl, nbls)
 
+    elif less2048:
+        message = (
+          "antnums_to_baseline: found antenna numbers > 255, using "
+          "2048 baseline indexing. Beware compatibility "
+          "with CASA etc"
+        )
```

Comment:
I don't think we need this message or the similar one below anymore. To get to CASA, users can use the `write_ms` method and uvfits and other formats have support for more than 256 antennas.

Reply:
unless this warning is issued because the user set the `attempt256` flag.